test/unit/slurm/test_slurm_config_generator/test_generate_slurm_config_files/sample_input.json

```diff
@@ -0,0 +1,78 @@
+{
+    "cluster": {
+        "label": "default",
+        "default_queue": "multiple_spot",
+        "queue_settings": {
+            "multiple_spot": {
```

Comment:
Minor: I thought underscore is not allowed in queue name, might be nice to change this here